src/Data/Text/Internal/Unsafe/Shift.hs

```diff
@@ -1,8 +1,5 @@
 {-# LANGUAGE CPP #-}
```

Comment:
Can't we get rid of this whole file? It says it's a workaround for GHC 6... I'd be fine punting this as a later TODO though.

Reply:
I'd say let's move the bigger code-change work to a follow up to unblock @Bodigrim